6-data-storage/03-indexeddb/article.md

```diff
@@ -103,9 +103,9 @@ let deleteRequest = indexedDB.deleteDatabase(name)
 ```warn header="We can't open an older version of the database"
 If the current user database has a higher version than in the `open` call, e.g. the existing DB version is `3`, and we try to `open(...2)`, then that's an error, `openRequest.onerror` triggers.
 
-That's odd, but such thing may happen when a visitor loaded an outdated JavaScript code, e.g. from a proxy cache. So the code is old, but his database is new.
+That's rare, but such a thing may happen when a visitor loads outdated JavaScript code, e.g. from a proxy cache. So the code is old, but his database is new.
 
-To protect from errors, we should check `db.version` and suggest him to reload the page. Use proper HTTP caching headers to avoid loading the old code, so that you'll never have such problem.
+To protect from errors, we should check `db.version` and suggest he reloads the page. Use proper HTTP caching headers to avoid loading the old code, so that you'll never have such problems.
```

Comment:
```suggestion
To protect from errors, we should check `db.version` and suggest a page reload. Use proper HTTP caching headers to avoid loading the old code, so that you'll never have such problems.
```

Reply:
Actually, I am not quite sure if your usage is grammatically correct. Something just doesn't feel right. I am not sure, so you could be right.

Reply:
Yes, you are right, it's a subjunctive. Glad you caught it.